diff -ur Clp-1.14.7/Clp/src/Makefile.am Clp-1.14.7-new/Clp/src/Makefile.am
--- Clp-1.14.7/Clp/src/Makefile.am	2011-06-19 18:27:26.000000000 +0200
+++ Clp-1.14.7-new/Clp/src/Makefile.am	2012-11-09 02:34:31.208611098 +0100
@@ -95,6 +95,8 @@
 # This is for libtool
 libClp_la_LDFLAGS = $(LT_LDFLAGS)
 
+libClp_la_LIBADD = $(CLPLIB_DEPENDENCIES)
+
 ########################################################################
 #                            clp program                               #
 ########################################################################
diff -ur Clp-1.14.7/Clp/src/Makefile.in Clp-1.14.7-new/Clp/src/Makefile.in
--- Clp-1.14.7/Clp/src/Makefile.in	2011-06-19 18:27:26.000000000 +0200
+++ Clp-1.14.7-new/Clp/src/Makefile.in	2012-11-09 02:34:09.156408715 +0100
@@ -83,7 +83,7 @@
 	"$(DESTDIR)$(includecoindir)"
 libLTLIBRARIES_INSTALL = $(INSTALL)
 LTLIBRARIES = $(lib_LTLIBRARIES)
-libClp_la_LIBADD =
+libClp_la_LIBADD = $(CLPLIB_DEPENDENCIES)
 am__libClp_la_SOURCES_DIST = ClpConfig.h ClpCholeskyBase.cpp \
 	ClpCholeskyBase.hpp ClpCholeskyDense.cpp ClpCholeskyDense.hpp \
 	ClpConstraint.cpp ClpConstraint.hpp ClpConstraintLinear.cpp \
